client/src/nonblocking/rpc_client.rs

```diff
@@ -5016,6 +5016,21 @@ impl RpcClient {
         .await
     }
 
+    pub async fn get_token_largest_accounts(
```

Comment:
Let's follow the model of the other `get_token_*` calls, and split it up to `get_token_largest_accounts` and `get_token_largest_accounts_with_commitment`

Reply:
Thanks for the suggestion. I went ahead and split them up.